modules/giapi/src/main/scala/giapi/client/gpi/GpiClient.scala

```diff
@@ -98,22 +110,67 @@ final class GpiClient[F[_]](override val giapi: Giapi[F]) extends GiapiClient[F]
       ),
       DefaultCommandTimeout
     )
+
+  override def genericApply(configuration: Configuration): F[CommandResult] = {
+    // TODO Implement a smarter apply
+    def smartApply(): F[CommandResult] =
+      giapi.command(Command(
+                      SequenceCommand.APPLY,
+                      Activity.PRESET_START,
+                      configuration
+                    ),
+                    DefaultCommandTimeout)
+
+    for {
+      _ <- statusDb.value("gpi:fpu") // placeholder
+      a <- smartApply()
+    } yield a
```

Comment:
This can be `statusDb.value("gpi:fpu") *> smartApply`. Not a big deal.

Reply:
Yeah. this is placeholder only, to avoid complains that `statusDb` isn't being called